README.md

```diff
@@ -55,53 +114,68 @@ git clone git@github.com:<username>/nuxt-i18n.git modules/nuxt-i18n
 }
 ```
 
-- Install the module's dependencies
+- Install the module's dependencies:
 
 ```sh
 cd modules/nuxt-i18n/
 yarn
 ```
 
+- Create a new branch for your changes:
+
+```sh
+git checkout -b my-new-feature
+```
+
 - Code, code, code!
-- Run ESLint & Jest
+- Run ESLint and Jest
 
 ```sh
 yarn lint && yarn test
 ```
 
+- Commit and push your changes
 - Submit awesome PRs
 
-## Contribute to the documentation
+### Documentation
 
-- Fork and clone the project
+- Fork and clone the project:
 
 ```sh
-git clone git@github.com:<username>/nuxt-i18n.git
-# or
-git clone https://github.com/<username>/nuxt-i18n.git
+git clone git@github.com:<username>/nuxt-i18n.git # ssh
+git clone https://github.com/<username>/nuxt-i18n.git # https
 ```
 
-- Install the dependencies
+- Install the dependencies:
 
 ```sh
-cd nuxt-i18n/
-yarn
+cd nuxt-i18n
+yarn # yarn
+npm i # npm
```

Comment:
Maybe let's skip whole clone/install deps steps for `Documentation` header since those are already listed for the project above?

Reply:
Then it would be unclear which of these steps apply to contributing to documentation and which don't.

Reply:
But it's all the same repo. There is no difference in contributing to code or documentation really, apart from running `docs:dev` script to develop documentation.

But if you are insisting, I won't hold it against this review. It's not that important.

If keeping it, there is a bit of incosistency as it mentions `npm` step but other places only mention `yarn`.